sources/packages/web/package-lock.json

```diff
@@ -30049,4 +30049,4 @@
       }
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
If there is no PR-related change please avoid adding it to the PR.

Reply:
It got added added during the git add command execution.
Never thought it will be like this 😊